CHANGELOG.md

```diff
@@ -43,33 +52,6 @@ user IP addresses, if you set `sendDefaultPii: true` in your `Sentry.init` optio
 
 We apologize for any inconvenience caused!
 
-- **feat(node): Add `ignoreStaticAssets` ([#17370](https://github.com/getsentry/sentry-javascript/pull/17370))**
```

Comment:
m: why was this removed? (rebase conflict?)

update: I see we moved the `ignoreStaticAssets` logic to node-core but isn't the user-facing change still the same?

Reply:
oops, that was not on purpose, will revert!